src/pages/api/init.js

```diff
@@ -1,14 +1,16 @@
 import fs from 'fs'
 
 export default function handler(req, res) {
-    console.log(req.body);
-    console.log(process.cwd())
-    const fileName = req.body.path.split('/').slice(-1)[0]
-    let fileArray = fileName.split('-')
-    fileArray = fileArray.map((word) => {return `${word.slice(0, 1).toUpperCase()}${word.slice(1)}`})
-    const pascalName = fileArray.join('').slice(0, -8)
-    try {
-        fs.writeFileSync(`../../../../../../.oc/previews/${pascalName}Preview.oc.tsx`, `import React from 'react'
+  const fileName = req.body.path.split('/').slice(-1)[0]
```

Comment:
Why do we need init? Won't save file be called from editor automatically

Reply:
The init is called when a component is created for the first time using bit create. Subsequent edits are done using the save-file

Reply:
Okay